src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -855,13 +826,10 @@ class ShenandoahHeap : public CollectedHeap {
 public:
   inline RememberedScanner* card_scan() { return _card_scan; }
   void clear_cards_for(ShenandoahHeapRegion* region);
-  void dirty_cards(HeapWord* start, HeapWord* end);
-  void clear_cards(HeapWord* start, HeapWord* end);
```

Comment:
These aren't used, but in looking through the code for marking and clearing cards in `ShenandoahDirectCardMarkRememberedSet::mark_range_as_...` I realize we might want to see if we want to use vector instructions or use `memset`, when the range is sufficiently large (future work).

Reply:
That's a good idea!